test/integration/patients/patient/patient.js

```diff
@@ -1,16 +1,30 @@
+import { v4 as uuid } from 'uuid';
+
+import { getRelationship } from 'helpers/json-api';
+
+import { getPatient } from 'support/api/patients';
+import { getAction } from 'support/api/actions';
+import { stateDone } from 'support/api/states';
+
+const testPatientId = uuid();
+
 context('patient page', function() {
   specify('context trail', function() {
     cy
       .routesForPatientDashboard()
       .routeActions()
       .routePatient(fx => {
-        fx.data.id = '1';
-        fx.data.attributes.first_name = 'First';
-        fx.data.attributes.last_name = 'Last';
+        fx.data = getPatient({
+          id: testPatientId,
```

Comment:
I don't know that an id needs to be set?  `getPatient` will return an id that the visit can then use.

Reply:
I think that makes sense as well. I just pushed an updated commit...

Reply:
I'll work on making these same types of changes to the other PRs as well.